src/kokkos/plugin-PixelVertexFinding/kokkos/gpuClusterTracksByDensity.h

```diff
@@ -249,9 +200,34 @@ namespace KOKKOS_NAMESPACE {
         float errmax,   // max error to be "seed"
         float chi2max,  // max normalized distance to cluster
         const Kokkos::TeamPolicy<KokkosExecSpace>::member_type& team_member) {
-      clusterTracksByDensity(vdata, vws, minT, eps, errmax, chi2max, team_member);
+      Kokkos::abort(
+          "clusterTracksByDensityKernel: device kernel not supported in Kokkos (see clusterTracksByDensityHost)");
     }
 
+    void clusterTracksByDensityHost(Kokkos::View<ZVertices, KokkosExecSpace> vdata,
+                                    Kokkos::View<WorkSpace, KokkosExecSpace> vws,
+                                    int minT,       // min number of neighbours to be "seed"
+                                    float eps,      // max absolute distance to cluster
+                                    float errmax,   // max error to be "seed"
+                                    float chi2max,  // max normalized distance to cluster
+                                    const team_policy& policy) {
```

Comment:
Eventually (to support asynchronous execution) the function signature would needs to be changed to something along
```suggestion
                                    const KokkosExecSpace& execSpace,
                                    const team_policy& policy) {
```
to be pass the same execution space object to `Hist::finalize()`. I'd prefer to do it now rather than later.

Same for the other `clusterTracksBy*Host()` functions.

Reply:
Done.